etc/helm/pachyderm/templates/kube-event-tail/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.kubeEventTail.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: kube-event-tail
+    suite: pachyderm
+  name: kube-event-tail
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kube-event-tail
+      suite: pachyderm
+  template:
+    metadata:
+      labels:
+        app: kube-event-tail
+        suite: pachyderm
+    spec:
+      containers:
+        - name: kube-event-tail
+          env:
+            - name: DEBUG_ADDRESS
+              value: 0.0.0.0:8081
```

Comment:
BTW, annoying "fun fact" here... the kubelet is actually doing those http readiness probes, not the pod itself, so debug has to be exposed to the network in order for it to get at healthz.  Very annoying, if you ask me :D